test/Tests/src/Data/Ordering_Spec.enso

```diff
@@ -36,9 +47,33 @@ Comparable.from (_ : UPair) = UPair_Comparator
 type Parent
     Value child
 
-# === The Tests ===
+# Just a type without custom comparator
+type No_Comp_Type
+    Value val
+
 
+## Expects that `result` contains incomparable values warning.
+   The values within the warning message can be switched - the order
+   does not matter. Iterates through all the warnings of result.
+expect_incomparable_warn : Any -> Any -> Any -> Nothing
+expect_incomparable_warn left_val right_val result =
+    # Incomparable values warning wraps Text values in simple quotes
+    left_val_text = if Meta.is_a left_val Text then "'" + left_val + "'" else left_val.to_text
+    right_val_text = if Meta.is_a right_val Text then "'" + right_val + "'" else right_val.to_text
+    expected_warn_msg_left = "Values " + left_val_text + " and " + right_val_text + " are incomparable"
+    expected_warn_msg_right = "Values " + right_val_text + " and " + left_val_text + " are incomparable"
+    has_expected_warning = Warning.get_all result . map (_.value) . any (it-> it == expected_warn_msg_left || it == expected_warn_msg_right)
+    has_expected_warning . should_be_true
+
+expect_no_warns : Any -> Nothing
+expect_no_warns result =
+    Warning.get_all result . length . should_equal 0
+
+
+# === The Tests ===
 spec =
+    topo_sort_pending = "Waiting for implementation of topological sort (https://github.com/enso-org/enso/issues/5742)"
```

Comment:
```suggestion
    topo_sort_pending = "Waiting for implementation of topological sort (https://github.com/enso-org/enso/issues/5834)"
```

Shouldn't we link to the [topo-sort issue](https://github.com/enso-org/enso/issues/5834) instead of [the current one, attached to this PR](https://github.com/enso-org/enso/issues/5742)?

Reply:
Definitely. I made a type. thx.